worker/backup_ee.go

```diff
@@ -20,7 +20,7 @@ import (
 	"time"
 
 	"github.com/golang/glog"
-	"github.com/golang/snappy"
+	"github.com/klauspost/compress/snappy"
```

Comment:
For backup, we could move to using s2 given s2 is backward compatible with snappy.